tests/MoBi.R.Tests/Services/SimulationTaskSpecs.cs

```diff
@@ -1,106 +1,100 @@
-﻿using System;
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
 using System.Linq;
 using MoBi.Core.Domain.Model;
 using MoBi.HelpersForTests;
-using MoBi.R.Domain;
 using MoBi.R.Services;
-using NUnit.Framework;
 using OSPSuite.BDDHelper;
 using OSPSuite.BDDHelper.Extensions;
+using OSPSuite.Core.Domain;
 using OSPSuite.R.Domain;
 using ModuleConfiguration = MoBi.R.Domain.ModuleConfiguration;
 
 namespace MoBi.R.Tests.Services
 {
    internal abstract class concern_for_SimulationCreateTask : ContextForIntegration<ISimulationTask>
    {
+      protected MoBiProject _project;
+      protected IProjectTask _projectTask;
+      protected IModuleTask _moduleTask;
+      protected string _simulationName = "Sim1";
+
       public override void GlobalContext()
       {
          base.GlobalContext();
          sut = Api.GetSimulationTask();
+         _projectTask = Api.GetProjectTask();
+         _moduleTask = Api.GetModuleTask();
+      }
+
+      protected void LoadSampleProject()
```

Comment:
I'd give the file name as an argument here so that this can be used again in the future with another file.